manifests/server.yml

```diff
@@ -43,8 +43,8 @@ instance_groups:
             username: management
             password: (( vault meta.vault "/rabbitmq/admin/management:password" ))
           broker:
-            username: broker
-            password: (( vault meta.vault "/rabbitmq/admin/broker:password" ))
+            username: (( grab meta.broker.username ))
+            password: (( grab meta.broker.password ))
```

Comment:
leave these as they were. These meta properties you're grabbing refer to the authentication to talk to the Broker's API, but they should be the creds to talk to the RabbitMQ as the broker's user for RMQ.